### PR TITLE
Turn the anonymous method to a concrete method for UWP.[No merge]

### PR DESCRIPTION
**Pull request by tijoytom**
ILC tool chain reducer do not play well with anonymous lambda used in reflection context ,  make it a public method.With this 10+ tests now pass.

### CODE REVIEW

**Comment by tijoytom** [comment]
@danmosemsft @joshfree @AtsushiKan

**Comment by danmosemsft** [comment]
Why doesn't it accept anonmymous lambda? That's a pretty mainstream scenario for customer code. Is there work tracked to address that?

@jkotas

**Comment by tijoytom** [comment]
@morganbr I think morgan mentioned it was broken with NS2.0 and he's working or going to work on it

File: src/System.Diagnostics.Process/tests/ProcessTestBase.cs [hunk 1]
```diff
@@ -39,9 +39,14 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
         }
 
+        public int SuccessFullExit()
```
Comment (danmosemsft) [inline src/System.Diagnostics.Process/tests/ProcessTestBase.cs]: typo should be as `SuccessfulExit`

**Comment by danmosemsft** [comment]
I don't object to making this specific test case change for now, to get tests running in the near term, but I don't think we should be changing many of these, especially in product code: we should fix ILC.

**Comment by tijoytom** [comment]
Bug here 
https://github.com/dotnet/corefx/issues/19249

**Comment by morganbr** [comment]
I'm not aware of any problems with anonymous methods. I'm only aware of a minor dynamic keyword issue that's unrelated to anonymous methods. Please investigate this further rather than changing the test.

**Comment by stephentoub** [comment]
> Please investigate this further rather than changing the test.

+1

**Comment by tijoytom** [comment]
Here is the pattern 
"() => SuccessExitCode)" is passed to RemoteInvoke method which does a MethodInfo on it and then invoke . The MethodInfo is null ... I should have said reflecting on anonymous method.  Now if you say analyzer can handle this , i can investigate further other i would concentrate on bigger failure buckets

**Comment by tijoytom** [comment]
Closing this since this need further investigation

**Pull request closed without merging**